packages/api/src/schema/schema.yaml

```
@@ -415,46 +457,63 @@ components:
         isActive:
           type: boolean
           description: If currently active
```

I think this one could be improved a bit wdyt? maybe just an expanded
```suggestion
          description: Whether this stream is currently active
```